purchases/src/test/java/com/revenuecat/purchases/PurchasesCoroutinesTest.kt

```diff
@@ -12,7 +12,7 @@ import org.robolectric.annotation.Config
 @OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class, ExperimentalCoroutinesApi::class)
 @RunWith(AndroidJUnit4::class)
 @Config(manifest = Config.NONE)
-class PurchasesCoroutinesTest : BasePurchasesTest() {
+internal class PurchasesCoroutinesTest : BasePurchasesTest() {
```

Comment:
Hmm I think for the tests we don't really care if they are public? Same for a bunch of others here.

Reply:
I can't make these public because they use internal classes and I get a compiler error saying that this class would expose those internal classes. Maybe there's another way that is not making these internal

Reply:
Oh interesting... This is ok then 👍